scripts/build/config.js

```diff
@@ -108,7 +108,7 @@ const pluginFiles = [
         module: require.resolve("flow-parser"),
         process(text) {
           const { fsModuleNameVariableName } = text.match(
-            /,(?<fsModuleNameVariableName>\w+)="fs",/,
+            /,(?<fsModuleNameVariableName>[\p{ID_Start}_$][\p{ID_Continue}$]*)="fs",/u,
```

Comment:
New `fsModuleNameVariableName` is `z$`. It isn't matched by `\w+`/. So I updated it makes to match all JS identifiers with Unicode property escape.

Reply:
ref: https://tc39.es/ecma262/#prod-IdentifierStartChar